public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -153,6 +157,44 @@ export function runSplitGroupedQueries(datasource: LokiDatasource, requests: Lok
   return response;
 }
 
+function updateLoadingFrame(
+  response: DataQueryResponse,
+  request: DataQueryRequest<LokiQuery>,
+  partition: TimeRange[],
+  requestN: number
+): DataQueryResponse {
+  if (isLogsQuery(request.targets[0].expr) || isLogsVolumeRequest(request)) {
```

Comment:
do we really need to skip histograms from this?

Reply:
i see. we can totally merge this approach, but i'd recommend checking what's wrong with the histogram. the progress-display would be very nice there too, i think.

Reply:
Yeah! That'd be a great follow up to this. Pinging @svennergr in case he's interested on adding this missing support 😅